File: 151002/Saprin/RV/publicator/src/main/java/by/bsuir/publicator/service/sticker/IStickerService.java
package by.bsuir.publicator.service.sticker;

import by.bsuir.publicator.dto.StickerRequestTo;
import by.bsuir.publicator.dto.StickerResponseTo;
import by.bsuir.publicator.exception.EntityNotFoundException;

import java.math.BigInteger;
import java.util.List;

public interface IStickerService {
    List<StickerResponseTo> getStickers();
    StickerResponseTo getStickerById(BigInteger id) throws EntityNotFoundException;
    StickerResponseTo createSticker(StickerRequestTo sticker);
    StickerResponseTo updateSticker(StickerRequestTo sticker) throws EntityNotFoundException;
    void deleteSticker(BigInteger id) throws EntityNotFoundException;

}

File: 151003/Klimovich/Publisher/Services/Interfaces/IStickerService.cs
using DC.Models.DTOs.ResponceTo;
using Publisher.Models.DTOs.DTO;

namespace Publisher.Services.Interfaces
{
    public interface IStickerService
    {
        StickerResponceTo CreateSticker(StickerRequestTo item);
        List<StickerResponceTo> GetStickers();
        StickerResponceTo GetSticker(int id);
        StickerResponceTo UpdateSticker(StickerRequestTo item);
        int DeleteSticker(int id);
    }
}
